jira_agile_metrics/calculators/defects.py

```diff
@@ -77,120 +90,198 @@ def run(self):
 
     def write(self):
         chart_data = self.get_result()
+        # fixversion_field_choice = self.settings['defects_fixversion_label']
+        # print(fixversion_field_choice)
+        x_label=self.settings['defects_fixversion_label']
         if chart_data is None:
             return
 
         if len(chart_data.index) == 0:
             logger.warning("Cannot draw defect charts with zero items")
             return
-        
+
         if self.settings['defects_by_priority_chart']:
-            self.write_defects_by_priority_chart(chart_data, self.settings['defects_by_priority_chart'])
-        
+            self.write_defects_by_priority_chart(chart_data, self.settings['defects_by_priority_chart'],x_label)
```

Comment:
There should be a space after the comma (several more examples of this below)

Reply:
I will fix that.